extensions/adobe/experience/adcloud/conversionmodel.schema.json

```diff
@@ -0,0 +1,160 @@
+        },
+        "xdm:clickThroughCount": {
+          "title": "Click Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Count for attributed models if trigger event is Click"
+        },
+        "xdm:clickThroughValue": {
+          "title": "Click Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Value for attributed models if trigger event is Click"
+        },
+        "xdm:viewThroughCount": {
+          "title": "View Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Count for attributed models if trigger event is Impression"
+        },
+        "xdm:viewThroughValue": {
+          "title": "View Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Value for attributed models if trigger event is Impression"
+        },
+        "xdm:clickviewThroughValue": {
+          "title": "Click View Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Value for attributed models if trigger event is click but impressions present in path"
+        },
+        "xdm:clickviewThroughCount": {
+          "title": "Click View Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Evaluated as Raw Conversion Count for attributed models if trigger event is click but impressions present in path"
+        },
+        "xdm:viewclickThroughValue": {
+          "title": "View Click Through Value Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Raw Conversion Value for attributed models if trigger event is impression but clicks present in path"
+        },
+        "xdm:viewclickThroughCount": {
+          "title": "View Click Through Count Model Attribution",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Raw Conversion Count for attributed models if trigger event is impression but clicks present in path"
+        },
+        "xdm:conversionType": {
+          "title": "Conversion Type",
+          "type": "integer",
+          "minimum": -32768,
+          "maximum": 32767,
+          "description": "Feed Conversion Type indicates where the conversion event originated from - examples client, analytics, cross device"
+        },
+        "xdm:device": {
+          "title": "Device",
+          "$ref": "https://ns.adobe.com/xdm/context/device",
+          "description": "The device from where the ad was displayed"
```

Comment:
Is it "The device where the conversion is originated" or "where the ad is displayed"?